plans/convert.pp

```diff
@@ -13,6 +13,9 @@
   # Common Configuration
   String                            $compiler_pool_address = $master_host,
   Array[String]                     $dns_alt_names         = [ ],
+
+  # Options
+  Boolean                           $configure_node_groups = true,
```

Comment:
Would like to see this dynamically calculated instead.  We can't trust humans to figure this out.